src/test/java/com/ldbc/driver/workloads/ldbc/snb/bi/BiWorkloadTest.java

```diff
@@ -298,6 +304,7 @@ public void shouldConvertFrequenciesToInterleavesWhenFrequenciesProvidedAndSomeI
                 .addOperationFrequency( LdbcSnbBiQuery22InternationalDialog.TYPE, 30 )
                 .addOperationFrequency( LdbcSnbBiQuery23HolidayDestinations.TYPE, 40 )
                 .addOperationFrequency( LdbcSnbBiQuery24MessagesByTopic.TYPE, 50 )
+                .addOperationFrequency( LdbcSnbBiQuery25WeightedPaths.TYPE, 50 ) // TODO
```

Comment:
TODO?

Reply:
Yes, but to be more specific: "TODO: where do these numbers come from?"

Reply:
I don't understand the question.
It is a test, those numbers are arbitrary in the `// given` part of the test.
What's important is the numbers in the `// then` section, which are computed based on these "arbitrary" numbers.